baggage/baggage.go

```diff
@@ -252,7 +252,14 @@ type Member struct {
 
 // NewMember returns a new Member from the passed arguments. An error is
 // returned if the created Member would be invalid according to the W3C
-// Baggage specification.
+// Baggage specification and an error will be returned if any of the input
+// parameters does not follow the specification.
```

Comment:
This is redundant.

```suggestion
// Baggage specification.
```

It might make more sense to generalize the reason an error is returned in the prior sentence fragment instead of restarting an almost identical idea here.

Reply:
noted, please check the latest version of these comments.